src/liquid/components/ld-label/readme.md

```diff
@@ -78,21 +78,56 @@ The default size is small. You can use a slightly bigger label (size medium) by
 #### Right
 
 {% example %}
-<ld-label position="right">
+<ld-label position="right" size="m">
   Email Address
   <ld-input placeholder="jane.doe@example.com" type="email"></ld-input>
 </ld-label>
 
 <!-- CSS component -->
 
-<label class="ld-label ld-label--right">
+<label class="ld-label ld-label--right ld-label--m">
   Email Address
   <div class="ld-input">
     <input placeholder="jane.doe@example.com" type="email">
   </div>
 </label>
 {% endexample %}
 
+### HTML Content
+
+{% example %}
+<ld-label position="right">
+  <span>I love to <code style="line-height: 0">code</code>.</span>
```

Comment:
I'd expect a developer might also want to add this label without the surrounding `<span>`-Element. Also what, if I want to add more than one element before (and/or after) the input element in question?

Reply:
The wrapping element is necessary. I have documented this though in my last commits. It should be fine as this is an edge case scenario anyway.

Reply:
Regarding more than one element before and or after: I currently can not think of any use case. In any case this seems to be even more an edge case which I would ignore for now.